addons/bestja_detailed_reports/views/detailed_reports_summary.xml

```diff
@@ -65,6 +67,21 @@
                 ('detailed_report.state', '!=', 'draft'),
             ]</field>
             <field name="context">{
+                'search_default_responsible_project': active_id,
+                'search_default_accepted': True,
+            }</field>
+        </record>
+
+        <record model="ir.actions.act_window" id="action_detailed_reports_product_summary_lvl0">
```

Comment:
They're not the same - one uses responsible_project and the other one top_project. The middle level organization needs to see its tonnage in the statistics, as well as the lower level organizations tonnage (hence uses responsible_project field). On the other hand, the top level organization wants to see only the sums for middle level organizations (for middle level organization sum of tonnage of this organization and its children; hence top_project).